packages/vitest/src/node/plugins/utils.ts

```diff
@@ -32,3 +32,39 @@ export function resolveOptimizerConfig(testOptionc: DepsOptimizationOptions | un
   }
   return newConfig
 }
+
+export function deleteDefineConfig(viteConfig: ViteConfig) {
+  const defines: Record<string, any> = {}
+  if (viteConfig.define) {
+    delete viteConfig.define['import.meta.vitest']
+    delete viteConfig.define['process.env']
+    delete viteConfig.define.process
```

Comment:
We should probably also remove `global`.

And having a test would be nice 😄 I think adding `define` in `test/core` and checking the value in a test should be fine.

Reply:
ok, I have updated the code. 😄